packages/cashier/src/Containers/__tests__/cashier.spec.js

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { isMobile } from '@deriv/shared';
+import Cashier from '../cashier';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+
+    return {
+        ...original_module,
+        Loading: jest.fn(() => 'mockedLoading'),
+    };
+});
+
+jest.mock('@deriv/shared', () => {
+    const original_module = jest.requireActual('@deriv/shared');
+
+    return {
+        ...original_module,
+        isMobile: jest.fn(() => false),
+        WS: {
+            wait: (...payload) => {
+                return Promise.resolve([...payload]);
+            },
+        },
+    };
+});
+
+jest.mock('Components/account-prompt-dialog', () => jest.fn(() => 'mockedAccountPromptDialog'));
+jest.mock('Components/error-dialog', () => jest.fn(() => 'mockedErrorDialog'));
+
+describe('<Cashier />', () => {
+    const props = {
+        is_account_setting_loaded: true,
+        is_account_transfer_visible: true,
+        is_logged_in: true,
+        is_payment_agent_transfer_visible: true,
+        is_payment_agent_visible: true,
+        is_p2p_enabled: true,
+        is_onramp_tab_visible: true,
+        is_visible: true,
+        location: {
+            pathname: '/cashier/deposit',
+        },
+        routes: [
```

Comment:
@farrah-deriv What if we can have mock data for the `routes-config` file and use it here (can reduce this file lines also)?